release/release.go

```diff
@@ -904,9 +877,9 @@ func uploadRelease(v version.Version) {
 				artifactURL := path.Join("fastdl.mongodb.org/tools/db", latestStableFile)
 
 				if ext == ".tgz" || ext == ".zip" {
-					download.Archive = toolsArchive{artifactURL, md5sum, sha1sum, sha256sum}
+					dl.Archive = download.ToolsArchive{artifactURL, md5sum, sha1sum, sha256sum}
 				} else {
-					download.Package = toolsPackage{artifactURL, md5sum, sha1sum, sha256sum}
+					dl.Package = &download.ToolsPackage{artifactURL, md5sum, sha1sum, sha256sum}
```

Comment:
nit: let's initialize the struct with keys instead of positional args. Positional struct initialization can be dangerous, especially when all fields are the same type, because the initialization can be silently invalidated if the struct definition is changed.